pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go

```diff
@@ -98,19 +94,22 @@ func NewClusterManagerController(
 	clusterManagerClient operatorv1client.ClusterManagerInterface,
 	clusterManagerInformer operatorinformer.ClusterManagerInformer,
 	deploymentInformer appsinformer.DeploymentInformer,
+	configMapInformer corev1informers.ConfigMapInformer,
 	recorder events.Recorder) factory.Controller {
 	controller := &clusterManagerController{
 		kubeClient:            kubeClient,
 		apiExtensionClient:    apiExtensionClient,
 		apiRegistrationClient: apiRegistrationClient,
 		clusterManagerClient:  clusterManagerClient,
 		clusterManagerLister:  clusterManagerInformer.Lister(),
+		configMapLister:       configMapInformer.Lister(),
 		currentGeneration:     make([]int64, len(deploymentFiles)),
 	}
 
 	return factory.New().WithSync(controller.sync).
 		ResyncEvery(3*time.Minute).
 		WithInformersQueueKeyFunc(helpers.ClusterManagerDeploymentQueueKeyFunc(controller.clusterManagerLister), deploymentInformer.Informer()).
+		WithInformersQueueKeyFunc(helpers.ClusterManagerConfigmapQueueKeyFunc(controller.clusterManagerLister), configMapInformer.Informer()).
```

Comment:
you would want to use a queukeyfunc with filter

Reply:
Done.